locales/en-US/app.ftl

```diff
@@ -446,6 +532,24 @@ TabBar--marker-table-tab = Marker Table
 TabBar--network-tab = Network
 TabBar--js-tracer-tab = JS Tracer
 
+## TrackContextMenu
+## This is used as a context menu for timeline to organize the tracks in the
+## analysis UI.
+
+TrackContextMenu--only-show-this-process-group = Only show this process group
+
+# This is used as the context menu item to show only the given track.
+# Variables:
+#   $trackName (String) - Name of the selected track to isolate.
+TrackContextMenu--only-show-track = Only show “{ $trackName }”
+TrackContextMenu--hide-other-screenshot-tracks = Hide other screenshot tracks
```

Comment:
Does "screenshot tracks" mean "screenshots of tracks" or something else? If that's correct, maybe "Hide screenshots of other tracks" would be clearer

Reply:
I wonder if "Screenshots tracks" is easier, given that's how they're called in the UI.

Reply:
Yeah, that would be better actually. Changed it.